app/plugins/batch_mode/batch_mode.h

```diff
@@ -62,7 +62,9 @@ class BatchMode : public BatchModeTags
 
 	vkb::FlagCommand categories_flag{vkb::FlagType::ManyValues, "category", "C", "Filter samples by categories"};
 
-	vkb::SubCommand batch_cmd{"batch", "Enable batch mode", {&duration_flag, &wrap_flag, &tags_flag, &categories_flag}};
```

Comment:
Should we add "S" as a shortcut here so it's similar to the other options?

Reply:
I don't think so.  While tags and categories have the short form, none of the other flags in any of the other plugins have a single-character form, and I suspect it would be too easy to accidentally assign the same short-form to more than one flag, such as the stretch flag or the screenshot flag, etc.

Reply:
Happy for this to be long form as the intended use case is for regression testing and not for a general user. In the future we may want to opt to use a json config to standardise this as our test framework over the existing system tests (which are not in use by Khronos but to my knowledge are still used by Arm)